src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Ball ball = new Ball();
+        ball.setColor(ColorSupplier.getRandomColor());
```

Comment:
The method 'getRandomColor()' is called as if it is a static method, but the checklist advises against using static methods. It's better to create an instance of 'ColorSupplier' and call the method on that instance.